backend.native/tests/codegen/coroutines/functionReference_eqeq_name.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package codegen.coroutines.functionReference_eqeq_name
+
+import kotlin.test.*
+
+suspend fun foo(x: Int) = x
+
+class Foo(val x: Int) {
+    suspend fun bar() = x
+}
+
+@Test fun runTest() {
+    val ref1 = ::foo
+    val rec = Foo(42)
+    val ref2 = rec::bar
+    val ref3 = ::foo
+    val ref4 = Foo(42)::bar
+    val ref5 = rec::bar
+    assertEquals("foo", ref1.name)
+    assertEquals("bar", ref2.name)
+    assertFalse(ref1 == ref2)
+    assertTrue(ref1 == ref3)
+    assertFalse(ref2 == ref4)
+    assertTrue(ref2 == ref5)
```

Comment:
As far as I see, test for `Foo::bar` is missing.

Reply:
What do you mean?

Reply:
I mean that `Foo::bar` is a proper callable reference too, and it is different from e.g. `Foo(42)::bar`. It makes some sense to check this too.